src/components/staking/PageValidator.vue

```diff
@@ -535,4 +528,10 @@ span {
     width: 50%;
   }
 }
+
+@media screen and (min-width: 1024px) {
```

Comment:
not a problem this time. but it is better to stick with either `max-width` or `min-width` so it is better to grasp what the breakpoints are.

In this case we could have done:
```
.validators-list-button {
  display: block;
}
@media screen and (max-width: 1024px) {
  display: none;
}
```